client/components/marketing-survey/cancel-jetpack-form/index.tsx

```diff
@@ -105,11 +105,11 @@ const CancelJetpackForm: React.FC< Props > = ( {
 	const offerDiscountBasedFromPurchasePrice = useMemo( () => {
 		if ( cancellationOffer ) {
 			// Get the percentage difference in the price of the offer over the price of the purchase.
-			// toFixed will round to the 3rd decimal place here.
-			// This will catch floats like .1999, and round them up.
+			// toFixed will round to the 2nd decimal place here.
+			// This will catch floats like .199, and round them up.
 			const offerPercentageOfPurchaseAmount = Number(
 				1 - cancellationOffer.rawPrice / purchase.amount
-			).toFixed( 3 );
+			).toFixed( 2 );
 			// Since toFixed returns a string, we use parseFloat here to get the decimal value.
 			// floor is used to get a whole percentage number value.
 			return Math.floor( Number.parseFloat( offerPercentageOfPurchaseAmount ) * 100 );
```

Comment:
The code that has been written here is interesting 🤔 Not sure what was the intention of rounding to the 3rd decimal place for numbers that are meant to be a percentage.

I don't understand also why `Math.floor` is used at the end of the function code, causing issues that we've just experienced.

Besides the solution you propose, I see also two alternatives, and the second one might be the most tempting.

1. We can replace `Math.floor` with `Math.round` to actually get the closes whole percentage rather than the lower value.
2. We can replace the code with one line `return Math.round( ( 1 - cancellationOffer.rawPrice / purchase.amount ) * 100 )` - In the end, `toFixed` is equivalent to `Math.round` operation.

Here is a link to playground that compares all solutions: [link](https://www.typescriptlang.org/play?#code/MYewdgzgLgBCBOBLA5osBDANgBScApjALwwCcAdKQKwDcAUKJLACaISgCuYU+zuiBYjAAslABz0G4aDAghMHKInABGIQAoAlMQB8MAN50YxmIxkAHfPALd0yQiQByHALYAjK+phqAtDFbsIFw8fHiEAPRwSKgYOGEwmuRQIABiiAAevF4ATAn0JjDw+FAc8GAwALLoUAAW5ABmmCAIXs7uVuTm6PAQ+ClN1V6W1vi29gkwAFTeAAwzeXQAvnRSTLLyispguSRaugZGJmawwzZQdg4wbR7wXr7+bJzcvPyCkQgoaFivhInJaZlmF4AMwLApFEplSrVOrwIJgIFXVw3TrdXr9ECDGCnUbncbaaYqOYLZarGRyBRKcCg3baIh6QwFY7YqxnC5Ce4BJ4hH4wd7RL5xAT4fImCGlcpVWrkOFcRE4saEQnEzT0Ulk+T4chNZDqADkAGUNlTyioAFwwPUAGnWlK2Kj2qo1mC1Ov1Rrt4Bg2Qt1ttm3A2Ud9DMmu1IF1huNWxgwN9NopAbAwODKyAA)